src/Kernel/Rectangle.class.st

```diff
@@ -751,10 +751,10 @@ Rectangle >> quickMerge: aRectangle [
 			[ useRcvr := false.
 			rCorner y ]
 		ifFalse: [ corner y ].
-	useRcvr
-		ifTrue: [ ^ self ]
+	^ useRcvr
+		ifTrue: [ self ]
 		ifFalse:
-			[ ^ Rectangle
+			[ self class
```

Comment:
i'm not sure about this change. CharacterBlock inherits from Rectangle.

Reply:
I have seen no sender of this method that seems to be on CharacterBlock.

IMO this is a good change because if we merge CharacterBlocks we expect a new character block. Same with other subclasses other people could do

Reply:
reverting change.